.changeset/empty-cheetahs-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/select": minor
+---
+
+Updating the `@kaizen/select` dependancies (specifically react-aria, shared-types and react-stately) caused a number of type errors that needed to be fixed. This related to the Node<T> generic that was being pulled in from shared-types that allowed a return value for null while our component filters out null values.
```

Comment:
```suggestion
Updating the `@kaizen/select` dependencies (specifically react-aria, shared-types and react-stately) caused a number of type errors that needed to be fixed. This related to the Node<T> generic that was being pulled in from shared-types that allowed a return value for null while our component filters out null values.
```

Although reading this, I'd probably expect the changelog to tell me something more along the lines of what was changed rather than the problem we came across.

Something like:
```
* Upgrade `react-aria`, `shared-types`, `react-stately`
  * Fix type errors where value can now be `null`
* Add type `MultiSelectItem` and replaced usage of `Node<T>`
```

Reply:
Fair point, I suppose anyone looking at this could refer to the PR to get the context if they really needed it